project_templates/technote_spietex/{{cookiecutter.series.lower()}}-{{cookiecutter.serial_number}}/spiebib.bst

```diff
@@ -0,0 +1,1108 @@
+% spiebib.bst  -  Version 3.33
```

Comment:
Consider pulling the version of spiebib.bst that I hacked in DMTN-229 so that it supports our `@Docushare` bib entries. Without that change they don't report the document handle or URL. Have we considered tracking these tex files in the lsst-texmf package? (I see some things are already in there) Or maybe we are not requiring lsst-texmf?

Reply:
i was thinking about addint to texxmf that but usually you have to submit a tar with all the stuff in it ... we cloud discuss. I believe i have exactly your spiebib is it not ?

Reply:
I had not realized that you knew about my hacked docushare. Yes, it is the right one (serves me right for not adding a change entry at the top of the file). SPIE takes the PDF and so does not care about the tex support files. arXiv has an old version of the class file, so you need to upload a newer version with the submission, but does not take bib style files (so it's moot).

Reply:
diff technote_spietex/\{\{cookiecutter.series.lower\(\)\}\}-\{\{cookiecutter.serial_number\}\}/spiebib.bst /Users/womullan/LSSTgit/dm-docs/DMTN/dmtn-229/spiebib.bst 
  
identical